### PR TITLE
Update contributing.md to include description of how to find build logs

### PR DESCRIPTION
**Pull request by gongmax**
Update contributing.md to include description of how to find build logs

### CODE REVIEW

File: contributing.md [hunk 1]
```diff
@@ -30,3 +30,13 @@ Guidelines](https://opensource.google/conduct/).
 All submissions, including submissions by project members, require review. We 
 use [GitHub pull requests](https://docs.github.com/articles/about-pull-requests)
 for this purpose.
+
+After creating a PR, a contributor needs to add `/gcbrun` to the comment to trigger CI
```
Comment (roberthbailey) [inline contributing.md:34]: nit: replace PR --> pull request

Comment (roberthbailey) [inline contributing.md:34]: can anyone leave the comment to trigger CI or does it need to be a maintainer? "contributor" sounds like the PR author.

Comment (roberthbailey) [inline contributing.md:34]: nit: "as a comment"

File: contributing.md [hunk 2]
```diff
@@ -30,3 +30,13 @@ Guidelines](https://opensource.google/conduct/).
...
+for the PR, which includes the CloudBuild triggered e2e test `pr-review-trigger` 
+and `pr-review-trigger-ap-clusters` as shown below. 
+![image](https://github.com/user-attachments/assets/6a759456-e1dd-4a3b-95da-84c9f2b8b053)
+
+If any of the build fails, you can find the link to the build log for these triggers at
```
Comment (roberthbailey) [inline contributing.md:39]: nit: "If any of the builds fail,"

File: contributing.md (hunk 1 above)
Comment (gongmax) [reply contributing.md:34]: The action itself says "Needs /gcbrun from a collaborator". So it's not everyone. I updated the word here to "collaborator".

**Review by roberthbailey — APPROVED** [review]

**Pull request merged**